src/util.rs

```diff
@@ -355,9 +438,7 @@ pub(crate) fn get_adapter_mtu(luid: &NET_LUID_LH) -> std::io::Result<usize> {
             }
         }
 
-        if let Err(e) = FreeMibTable(if_table as *mut _) {
```

Comment:
Still curious about this. I don't like merging if I cant understand why something changed. Perhaps this _is_ justified, but I currently don't know why.